src/featbin/compute-kaldi-pitch-feats.cc

```diff
@@ -37,8 +37,8 @@ int main(int argc, char *argv[]) {
         "compute-kaldi-pitch-feats --sample-frequency=8000 scp:wav.scp ark:- \n"
         "\n"
         "See also: process-kaldi-pitch-feats, compute-and-process-kaldi-pitch-feats\n";
-    
```

Comment:
there are blank spaces added at the ends of lines in this file and in others.  Can you please figure out how to automatically get rid of them?  You should change your editor settings too.